models/User.js

```diff
@@ -1,5 +1,102 @@
 const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
+
+//Do we want start dates required?
```

Comment:
I think we should keep dates optional

Reply:
I personally think that start dates should be mandatory for things like jobs and projects but not end dates as we need a way of handling whether or not they are still ongoing.

Reply:
That's a good point, I was reading this comment as if it was associated with `EducationSchema`. I think it should be optional for `EducationSchema` and required for `ExperienceSchema`